demo/demo/settings.py

```diff
@@ -89,7 +89,7 @@
 DATABASES = {
     'default': {
         'ENGINE': engine,
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'NAME': os.path.join('db.sqlite3'),
```

Comment:
Yeah. Actually, I'm going to set the right name for the test database. That was the issue that kinda forced me to remove `BASE_DIR`. When I launched unittests, Django was trying to create a file `test_/path/to/db.sqlite3`. I'll amend this.